wang1kevin1/InstitutoTerra
# Scratch/user profile

To Test:

- Need to re-signup. Please check that verification email comes from no-reply@refloresta.app
- Test user profile carousel
- test about page works on profile
- test that numbers change when trees planted (will need to checkout some trees)
- test that sharing works (tree message when sharing from trees, CO2 message when sharing from carbon)
